src/lib/scheduler.js

```diff
@@ -121,14 +121,14 @@ const getDefaultSemesterIndex = () => {
   const now = new Date();
   // Check if Winter (Period 3, above)
   if (
-    new Date(DATES.PREREG.WINTER) < now &&
-    now < addDays(new Date(DATES.Winter.END), -7)
+    new Date(gcalFormattedDate(DATES.PREREG.WINTER)) < now &&
+    now < new Date(gcalFormattedDate(DATES.Winter.START))
```

Comment:
Good work. I never noticed that this function exists...

One slight thing. Maybe rename this function to something like `RFC3339FormattedDate` or `JavaScriptFormattedDate`? Since the function returns RFC3339-formatted date, not ICS/gCal formatted dates...